mesonbuild/wrap/wrap.py

```diff
@@ -427,6 +427,24 @@ def find_program_provider(self, names: T.List[str]) -> T.Optional[str]:
                 return wrap_name
         return None
 
+    def get_cargo_wraps(self, cratename: str) -> T.List[str]:
+        found = []
+        for wrap in self.wraps.values():
+            if wrap.get('method') != 'cargo':
+                continue
```

Comment:
Unless I'm missing something, `self.wraps` is a `dict[str, PackageDefinition]` and `PackageDefinition` does not have a `get()` method. How is this supposed to work?